Add a funtion to forge Android logd entry

DimaWittmann opened the pull request:
This PR adds a function that constructs a custom log entry.
I can be used it forge log entries, for example, to forward logs for 3rd party app to lodg

Commit:
Add a funtion to forge Android logd entry

flxo commented on src/lib.rs:
````diff
@@ -63,6 +63,12 @@
+//! To forge android logd entry:
+//!
+//! ```
+//! android_logd_logger::low(buffer_id, pid, thread_id, secs, subsec_nanos, tag, priority, message).unwrap();
````
Here's a typo. Should be `log`.  Can you please also ensure that rustdoc catches the examples and tries to build them?

DimaWittmann replied:
Thank you for suggestion. I checked with the rustdoc

flxo commented on src/lib.rs:
```diff
@@ -490,3 +513,48 @@ impl Builder {
+    crate::logd::log(&record);
+
+    #[cfg(not(target_os = "android"))]
+    println!("buffer: {:?}, tag: {:?}, message: {:?}", buffer_id, tag, message);
```
Please forward the records in the not android case as well. This keeps the equals format.

DimaWittmann replied:
Of course, I was just not sure, I removed this conditional compilation

flxo reviewed (changes requested):
Thanks for your contribution. Left some comments.

Commit:
Remove chrono dependency and use SystemTime

Remove the additional heavy dependency `chrono` by using
`std::time::SystemTime` instead of secs and subsecs.

Commit:
Reuse formatting for non android systems

Commit:
Use SystemTime in Record
